app/src/main/java/edu/uc/zhukv/droneradarmap/DetailsFragment.kt

```diff
@@ -26,4 +22,7 @@ class DetailsFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
```

Comment:
Changed order of contents of class
https://kotlinlang.org/docs/coding-conventions.html#class-layout